### ISSUE TITLE
TFRC is not a peptidase

### ISSUE THREAD

**Issue by RLovering** [geneontology/go-annotation#3797]
* **IPR036757 Transferrin receptor-like, dimerisation domain superfamily **

* **Example sequences with problematic annotation (TFRC P02786 transferrin receptor protein 1 ):**

* **Description of issue**
This family page description states: ... Not all members of this subfamily are peptidases, and non-peptidase homologues include transferrin receptor proteins. 
And yet TFRC P02786 transferrin receptor protein 1 is associated by InterPro with the GO term GO:0008235 metalloexopeptidase activity which lead the GOC inference of GO:0006508 | proteolysis
This association is due to the membership of TFRC to the IPR007484 Peptidase_M28 family.

I am not sure how you can stop the propagation of these terms to family members that do not have the activity of the family but it would be great if you could add this as an exception, rather than removing the term from IPR007484 Peptidase_M28.

While you are there you might want to add the GO BP term  GO:0006508 | proteolysis to the IPR007484 Peptidase_M28 record so that InterPro gets credit for this assertion rather than the GOC.

Many thanks

Ruth

**Comment by sarach06** [comment]
Hi,
Thank you very much. I have removed the term from IPR007484 as it includes non-peptidase homologues. I created a new entry with PTHR12147 which includes a set of M28 peptidases (subfamilies A, C, E, and F) and added the terms GO:0008235 and GO:0006508 (this will be available in the next InterPro release 87.0). Together with PTHR10404 (IPR039373) and PTHR12053 (IPR039866), the majority of M28 family members with peptidase activity will be covered. Let me know if you agree.

Best regards,
Sara
